package.json

```diff
@@ -78,10 +80,14 @@
     "@babel/plugin-proposal-object-rest-spread": "^7.16.0",
     "@babel/preset-env": "^7.16.4",
     "@babel/preset-typescript": "^7.16.0",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
+    "react": "^18.1.0",
+    "react-dom": "^18.1.0",
     "wsrun": "^5.2.0"
   },
+  "resolutions": {
+    "@types/react": "^18.0.9",
+    "@types/react-dom": "^18.0.5"
```

Comment:
Did you intend to do this?

Reply:
yep. without it seemed to get different versions which caused things to break

Reply:
Better to pin to 18.1 maybe?

Reply:
Dunno how the versioning is done but it's not the same, see https://www.npmjs.com/package/@types/react